include/inviwo/core/datastructures/volume/volume.h

```diff
@@ -73,7 +73,15 @@ class IVW_CORE_API Volume : public Data<Volume, VolumeRepresentation>,
                     const Wrapping3D& wrapping = wrapping3d::clampAll);
     explicit Volume(std::shared_ptr<VolumeRepresentation>);
     Volume(const Volume&) = default;
-    Volume(const Volume&, NoData);
+    /**
+     * Create a volume based on \p rhs without copying any data. If \p defaultFormat is a nullptr,
+     * the format of \p rhs is used.
+     * @param rhs             source volume providing the necessary information like dimensions,
+     *                        swizzle masks, interpolation, spatial transformations, etc.
+     * @param defaultFormat   data format of the new volume. If equal to nullptr, the format of \p
+     *                        rhs is used instead.
+     */
+    Volume(const Volume& rhs, NoData, const DataFormatBase* defaultFormat = nullptr);
```

Comment:
Potentially we could have optional arguments for all the parameters,
```suggestion
    Volume(const Volume& rhs, NoData, std::optional<size3_t> dims = {}, const DataFormatBase* defaultFormat = nullptr, std:optional<SwizzleMask> defaultSwissleMaks = {}, std::optional<InterpolationType> interpolation = {}, std::optional<Wrapping3D> wrapping = {});
```

I think it would be nice to have the same order of arguments as in the main constructor above...

Reply:
Implemented for Volume and Layer.